themes/default/content/blog/changes-to-import/index.md

```diff
@@ -0,0 +1,351 @@
+---
+title: "Overhaul of the `pulumi import` system"
```

Comment:
Why "Improving ..." vs something like "Enabling You to Import and Generate IaC for *ANY* Resource"

cc @gthuang @aaronkao 

Let's make it exciting! "Improving", insider baseball, etc, are boring ways to frame what is actually amazing work!

Reply:
Also cc @Brant-Soler on whether he has ideas on title/meta improvements for SEO 😄

Reply:
We could approach this entirely differently as a "re-launch" of the feature.  Basically pretend as though https://www.pulumi.com/blog/pulumi-import-generate-iac-for-existing-cloud-resources/ wasn't written, and write that blog post again relative to the new state of the feature.

Ultimately - there are three options:
1. Just update the existing blog post to match the current reality - and re-promote it.
2. Write a blog post about the improvements we've made (but emphasize the current state of the feature, not the diff)
3. Write a new blog post that re-announces the feature from scratch (effectively ignoring the previous post).

I was suggesting (2), but sounds like Joe sees (3) as a better approach.  Happy to go with that.

Reply:
The link (https://www.pulumi.com/blog/pulumi-import-generate-iac-for-existing-cloud-resources/) has a history with Google, and we can cash in on that. I recommend expanding on the article to:

1. Make it more exciting!
2. Include the past AND current state to focus on growth.
3. Include how we arrived at the current state and HOW that will inform a future state. 
4. Include more code snippets, these are important for catching how developers search for information.

A few questions to ignite creativity:

Is it possible for some developers to use the previous version if they choose? It's popular for certain editions of any piece of software to be chosen over other newer ones like versions of PHP.

Can you think of any external voices that would be willing to contribute? Testimonials are powerful in every form, if you can get a developer talking about why they make a certain choice, code with a certain pattern, or adhere to a certain style guide, other developers can relate to it.

One thing I'm seeing is brevity over depth in many areas of the website, which I completely understand for focus. I'm a fan of deep knowledge and extensive human-spoken English (transcripts for all videos should be available beneath the imported video tucked within an accordion) because of my SEO perspective.